plugins/inputs/statsd/statsd_test.go

```diff
@@ -133,13 +134,17 @@ func BenchmarkParser(b *testing.B) {
 		AllowedPendingMessages: 250000,
 		NumberWorkerThreads:    5,
 	}
+	parser := &graphite.Parser{Separator: plugin.MetricSeparator, Templates: plugin.Templates}
+	err := parser.Init()
+	require.NoError(b, err)
```

Comment:
Same for all other occurrences
```suggestion
	require.NoError(b, parser.Init())
```